### ISSUE TITLE
AWS CodeArtifact Package Group Configuration

### ISSUE THREAD

**Issue by ewbankkit** [hashicorp/terraform-provider-aws#36528]

**Comment by mungojam** [comment]
I'm going to have a go at this one as it would really streamline our package management. It's my first time working in Go and in my spare time, so I may not get anywhere for a while.

**Comment by mungojam** [comment]
I'm back on trying to get this one done while I've got some free evenings this week, unless anybody is already on it. Please shout if you are and I'll find another side project :)

**Comment by mungojam** [comment]
The motivation is going when there is the auto-generated awscc alternative which would likely work fine for us:

https://registry.terraform.io/providers/hashicorp/awscc/latest/docs/resources/codeartifact_package_group

**Comment by hotzen** [comment]
We are in the same boat, did you go awscc?

**Comment by mungojam** [comment]
> We are in the same boat, did you go awscc?

@hotzen yes, we did and it's been working fine for us. Would be nice to have it in the main provider though
